comments.php

```diff
@@ -27,23 +27,31 @@
 	if ( have_comments() ) : ?>
 		<h2 class="comments-title">
 			<?php
+			$comment_count = get_comments_number();
+			if ( 1 === $comment_count ) {
```

Comment:
This seems like a perfect instance where [_n()](https://codex.wordpress.org/Function_Reference/_n) should be used. Is this an oversight or was it intentionally avoided?

Reply:
Perhaps this can be improved. We're following _s on this: https://github.com/Automattic/_s/blob/master/comments.php and it passed review so for now, I'm happy to leave it to save time. If you're keen to submit a PR later, that's super.

Reply:
When I say it passed review, it got brought up, we changed to current function and then it passed, in other words, we've spent some time on this already.

Reply:
Weird stuff.